linux/sound.h

```diff
@@ -39,36 +39,37 @@
 #include "global.h"
 
 #if WITH_SOUND
-# include <jack/jack.h>
-# include <jack/midiport.h>
+#include <jack/jack.h>
+#include <jack/midiport.h>
 #endif
 
-
 /* Definitions ****************************************************************/
-#define NUM_IN_OUT_CHANNELS         2 // always stereo
+#define NUM_IN_OUT_CHANNELS 2 // always stereo
 
 // the number of periods is critical for latency
-#define NUM_PERIOD_BLOCKS_IN        2
-#define NUM_PERIOD_BLOCKS_OUT       1
-
-#define MAX_SND_BUF_IN              200
-#define MAX_SND_BUF_OUT             200
+#define NUM_PERIOD_BLOCKS_IN 2
+#define NUM_PERIOD_BLOCKS_OUT 1
 
+#define MAX_SND_BUF_IN 200
+#define MAX_SND_BUF_OUT 200
 
 /* Classes ********************************************************************/
 #if WITH_SOUND
 class CSound : public CSoundBase
 {
     Q_OBJECT
 
-public:
-    CSound ( void           (*fpNewProcessCallback) ( CVector<short>& psData, void* arg ),
+  public:
+    CSound ( void ( *fpNewProcessCallback ) ( CVector<short>& psData,
+                                              void*           arg ),
              void*          arg,
              const QString& strMIDISetup,
              const bool     bNoAutoJackConnect,
              const QString& strJackClientName ) :
         CSoundBase ( "Jack", fpNewProcessCallback, arg, strMIDISetup ),
-        iJACKBufferSizeMono ( 0 ), bJackWasShutDown ( false ), fInOutLatencyMs ( 0.0f )
+        iJACKBufferSizeMono ( 0 ),
```

Comment:
Can't blame the formatter for the mess here - the original could have been better.